src/jgikbase/idmapping/storage/id_mapping_storage.py

```diff
@@ -111,19 +111,17 @@ def remove_user_from_namespace(self, namespace_id: NamespaceID, admin_user: User
         raise NotImplementedError()
 
     @_abstractmethod
-    def set_namespace_publicly_mappable(
-            self,
-            namespace_id: NamespaceID,
-            publically_mappable: bool=False
-            ) -> None:
+    def set_namespace_publicly_mappable(self, namespace_id: NamespaceID, publically_mappable: bool
```

Comment:
What does publicly mappable mean exactly? You don't really define it in the design doc. Does it mean that anyone can create a mapping to/from it? Or does it mean something more like:

- To create a mapping from A to B, you have to be admin of A, and B has to be publicly mappable

Reply:
The implications of publicly mappable are laid out here: https://github.com/jgi-kbase/IDMappingService/blob/develop/design/design.md#creating-and-deleting-mappings